Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM registry.access.redhat.com/ubi7/ubi
```

Comment:
Image should ideally match between Dockerfile and Dockerfile-tools

The ones in https://github.com/IBM/nodejs-cloudant do